inspirehep/modules/migrator/tasks.py

```diff
@@ -288,9 +289,27 @@ def create_record(record):
     return json
 
 
+def attach_documents_and_figures(record):
+    control_number = record.get('control_number', record.get('recid'))
```

Comment:
If you are expecting the record to have at least one of those (control_number or recid) you should assert it's not none, if you are requiring it to have a recid, that can be done by just using `record['recid']` instead of using `get` there, so it fails if it does not have a recid.